Dockerfile.builder-go

```diff
@@ -1,4 +1,4 @@
 # this is used for testing jx inside a cluster in development
-FROM gcr.io/jenkinsxio/builder-go:latest
+FROM gcr.io/jenkinsxio/builder-go:0.0.0-SNAPSHOT-PR-1250-2
```

Comment:
what do we put there when we want to merge? Back to 'latest'?

Reply:
Yup.